src/krate.rs

```diff
@@ -1423,6 +1462,42 @@ pub fn owners(req: &mut Request) -> CargoResult<Response> {
     Ok(req.json(&R { users: owners }))
 }
 
+/// Handles the `GET /crates/:crate_id/owner_team` route.
+pub fn owner_team(req: &mut Request) -> CargoResult<Response> {
```

Comment:
We tried, but the structure of the response has `user:` vs `team:` change that ember cares about, and I suck at ember and I'm not sure why it cares about that. If we duplicate the response structure too, then it's most of the function that's still duplicated :-/

I wouldn't be opposed to refactoring this in the future, it would just take me more time in ember than we'd like to spend right now :P